radar-auth/src/main/java/org/radarcns/auth/authentication/TokenValidator.java

```diff
@@ -150,10 +150,9 @@ public RadarToken validateAccessToken(String token) throws TokenValidationExcept
                 }
                 return new JwtRadarToken(jwt);
             } catch (SignatureVerificationException sve) {
-                LOGGER.warn("Client presented a token with an incorrect signature, fetching public "
-                        + "keys again. Token: {}", token);
+                LOGGER.debug("Client presented a token with an incorrect signature. Token: {}",
```

Comment:
Perhaps it would be best to remove the token altogether: it leaks credentials to the logs.